index.js

```diff
@@ -116,12 +120,24 @@ exports.execute = function(config, callback) {
             return write(sheetFront, callback);
         },
         function(callback) {
+            return async.eachSeries(cols, function(col, callback) {
+                var colStyleIndex = col.styleIndex || 0;
+                var res = '<x:col min="' + cn + '" max="' + cn + '" width="' + col.width + '" customWidth="1" style="' + colStyleIndex + '"/>';
```

Comment:
Would it make sense to default the col.width in the event that it's not specified so that it does go to undefined.